### ISSUE TITLE
Duplicating worksheet

### ISSUE THREAD

**Issue by qtaa246** [gdemin/excel.link#14]
Hi

Is there any way to duplicate worksheet like [holding Crtl key + dragging worksheet tap]  ?

**Comment by gdemin** [comment]
You can use the following function:
```
xl.sheet.duplicate = function(before = NULL){
    ex = xl.get.excel()
    if (is.null(before)) {
        before = ex[['ActiveWorkbook']][['ActiveSheet']]
    } else {
        before = ex[['ActiveWorkbook']][['Sheets']][[before]]
    }
    ex[['ActiveWorkbook']][['Activesheet']]$copy(before = before)
    invisible(ex[['ActiveWorkbook']][['Activesheet']][['Name']])
}
```

**Comment by qtaa246** [comment]
Thank you !

**Comment by gdemin** [comment]
Fixed in version 0.9.8
